Gems/AzQtComponentsForPython/Code/license_header.txt

```diff
@@ -0,0 +1,8 @@
+/*
```

Comment:
This text file is unneeded since all it is an empty file with the O3DE copyright blerb.
The LICENSE.txt in the root of this repo covers the Apache 2.0 license for this gem

Reply:
This file is passed into the shiboken command line and is placed at the top of all files it generates. If this is unnecessary, will remove.